internal/cluster/distribution/eks_node_pool.go

```diff
@@ -48,7 +48,7 @@ func (n NewEKSNodePool) Validate() error {
 			violations = append(violations, "minimum autoscaling size cannot be lower than one")
 		}
 
-		if n.Autoscaling.MaxSize < n.Autoscaling.MinSize {
+		if n.Autoscaling.MaxSize <= n.Autoscaling.MinSize {
```

Comment:
If Autoscaling.MaxSize equals Autoscaling.MinSize then the cluster autoscaling loses it's function. IMHO if cluster autoscaler is enabled for a node pool then max size >= min size+1

Reply:
If the maxSize equals (or is less than) minSize validation fails. In other words, validation only passes when max size >= min size+1